specification/cognitiveservices/Speech.BatchTranscription/client.tsp

```diff
@@ -0,0 +1,31 @@
+import "./main.tsp";
+import "./models.tsp";
+import "@azure-tools/typespec-client-generator-core";
+
+using Azure.ClientGenerator.Core;
+
+namespace Azure.Ai.Speech.BatchTranscription;
+
+// TypeScript uses PascalCase for service names to match namespace conventions
+@@clientNamespace(Azure.Ai.Speech.BatchTranscription,
+  "Azure.AI.Speech.BatchTranscription",
+  "javascript"
+);
+
+// Client customizations for property names
+
+@@clientName(TranscriptionJob.contentUrls, "contents", "csharp/java");
+@@clientName(TranscriptionJob.contentContainerUrl,
+  "sourceContainer",
+  "csharp/java"
```

Comment:
I believe the correct thing here is to use individual `@clientName` decorators for java and dotnet.  Also, I think "dotnet" is the more appropriate designation for the csharp emitter (though it may allow both).